willsCheatMenu/src/index.ts

```diff
@@ -118,6 +124,9 @@ export class Toggler extends Hack {
 	}
 	setEnabled(event: () => unknown) {
 		this.enabled = event;
+		if (localStorage.getItem(this.name) === "true") {
```

Comment:
This should be called in the constructor, not in setEnabled

Reply:
If it is called in the constructor ``this.enabled`` will not be defined and the function will not execute.

Reply:
no

and where it is now doesn't do anything

Reply:
Okay fixed it.